frontend/amp/components/lib/AMPRenderer.tsx

```diff
@@ -0,0 +1,28 @@
+import { TextBlockComponent } from '../elements/TextBlockComponent';
+
+import React from 'react';
+
+export const AmpRenderer: React.SFC<{
+    elements: CAPIElement[];
+    pillar: Pillar;
+}> = ({ elements, pillar }) => {
+    const output = elements
+        .map((element, i) => {
+            switch (element._type) {
+                case 'model.liveblog.TextBlockElement':
+                    return (
+                        <TextBlockComponent
+                            key={i}
```

Comment:
Not sure how relevant it is in this instance, but React [advises against](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) using index as a key

Reply:
This meets the criterion for safely using index as `key` as we don't change this list, don't reorder it and the components don't have ids.